examples/networking/coap/nanocoap_server/Makefile

```diff
@@ -24,10 +24,11 @@ ifeq (,$(filter 1, $(LWIP_IPV4) $(LWIP_IPV6)))
 else
   ifeq (1,$(LWIP_IPV4))
     USEMODULE += ipv4_addr
-
     USEMODULE += lwip_arp
     USEMODULE += lwip_ipv4
-    USEMODULE += netdev_legacy_api
```

Comment:
But this is true here too :exploding_head:

Reply:
Thank for explanation for `netdev_legacy/new_api` - I remove it.